### PR TITLE
feat: new account switcher

### PR DESCRIPTION
**Pull request by meeh0w**
## Description
* [CP-9490](https://ava-labs.atlassian.net/browse/CP-9490)

## Changes
* New account switcher designs
* New account details view
* ~_No balance rollup yet (AC3), will be done in a separate PR_~
    * Done in #101 

## Testing
* Play with the account switcher, try to break it

## Screenshots:

https://github.com/user-attachments/assets/b639e4d8-3e80-4b83-b8ac-b683faebd8aa

## Checklist for the author
- [ ] I've covered new/modified business logic with Jest test cases.
- [x] I've tested the changes myself before sending it to code review and QA.

### CODE REVIEW

File: src/pages/Accounts/components/AccountsActionButton.tsx [hunk 1]
```diff
@@ -136,28 +137,34 @@ export const AccountsActionButton = ({
...
           display: 'flex',
           width: '100%',
-          mr: 0.5,
         }}
       >
         <Button
           onClick={onAddNewAccount}
-          sx={{ gap: 1 }}
+          sx={{
+            mr: '1px !important',
```
Comment (vvava) [inline src/pages/Accounts/components/AccountsActionButton.tsx:145]: Could we avoid to use the important here and other places?

**Review by vvava — CHANGES_REQUESTED** [review]
I've benn trying to test it as well, and it seems it works well, love it

File: src/pages/Accounts/components/AccountsActionButton.tsx (hunk 1 above)
Comment (meeh0w) [reply src/pages/Accounts/components/AccountsActionButton.tsx]: Refactored in [9633fa8](https://github.com/ava-labs/core-extension/pull/94/commits/9633fa819e6c336b0b5d7b443d86be05c66cd517)

**Review by bferenc — DISMISSED** [review]
works great! 👏

**Review by vvava — APPROVED** [review]
thank you!

**Comment by meeh0w** [comment]
# ⚠️ DO NOT MERGE, waiting for the balance rollup

**Review by vvava — APPROVED** [review]

**Review by bferenc — APPROVED** [review]

**Pull request merged**